cmd/virtual-kubelet/commands/root/tracing.go

```diff
@@ -98,5 +98,7 @@ func setupZpages(ctx context.Context) {
 	}
 	mux := http.NewServeMux()
 	zpages.Handle(mux, "/debug")
-	http.ListenAndServe(p, mux)
+	err := http.ListenAndServe(p, mux)
+	// This should never terminate
+	log.G(ctx).WithError(err).Fatal("Zpages server exited")
```

Comment:
Couple of things here...
- We should check for an error.
- I think ListenAndServe will always return an error unless something changed recently.
- Can this return an error instead of calling Fatal?

Reply:
👍

Reply:
If it always returns an error, why would we check for an error?